torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -244,13 +260,72 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
                         )
 
                     node.replace_all_uses_with(replacement_proxy.node)
-
                     graph.erase_node(node)
+                    deleted.add(node)
```

Comment:
Not sure why you need to track these, because you've already erased them from the graph